libs/portals/my-pages/health/src/screens/MedicinePrescriptions/MedicinePrescriptions.tsx

```
@@ -0,0 +1,159 @@
+      title={formatMessage(messages.medicinePrescriptions)}
+      intro={formatMessage(messages.medicinePrescriptionIntroText)}
+      serviceProviderSlug={HEALTH_DIRECTORATE_SLUG}
+      serviceProviderTooltip={formatMessage(
+        messages.landlaeknirMedicinePrescriptionsTooltip,
+      )}
+    >
+      <Box>
+        <Box display="flex" flexDirection="row" flexWrap={'wrap'}>
+          {tagLabels.map((item) => (
+            <Box marginRight={1} marginBottom={[1, 1, 1, 2, 2]}>
+              <Tag
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)